Fabric.Identity.API/Persistence/SqlServer/Models/ApiResourceDomainModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Fabric.Identity.API.Persistence.SqlServer.Models
+{
+    public class ApiResourceDomainModel : IdentityServer4.EntityFramework.Entities.ApiResource, ITrackable, ISoftDelete
```

Comment:
I think we might be better off calling this ApiResourceEntityModel, since it isn't a domain model.

Reply:
Yeah I thought the same thing and have started changing them over to be called entities